develop/parachain-devs/runtime-development/build-deterministic-runtime.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Build a deterministic runtime
+description: Explains how to use the Substrate runtime toolbox and Docker to build deterministic Wasm binaries for Polkadot SDK-based chains.
+---
+
+# Build a Deterministic Runtime
+
+## Introduction
+
+By default, the Rust compiler produces optimized Wasm binaries. These binaries are suitable for working in an isolated environment, such as local development. However, the Wasm binaries the compiler builds by default aren't guaranteed to be deterministically reproducible. Each time the compiler generates the Wasm runtime, it might produce a slightly different Wasm byte code. This is problematic in a blockchain network where all nodes must use exactly the same raw chain specification file.
+
+Working with builds that aren't guaranteed to be deterministically reproducible can cause other problems, too. For example, for automating the build processes for a blockchain, it is ideal that the same code always produces the same result (in terms of bytecode). Without a deterministic build, compiling Compiling the Wasm runtime with every push would produce inconsistent and unpredictable results without a deterministic build, making it difficult to integrate with any automation and likely to continuously break a CI/CD pipeline. Deterministic builds—code that always compiles to exactly the same bytecode—ensure that the Wasm runtime can be inspected, audited, and independently verified.
+
+## Prerequisites
+
+Before you begin, ensure you have the following:
+
+- [Docker](https://www.docker.com/get-started/){target=\_blank} installed
+- A [Docker account](https://app.docker.com/signup/){target=\_blank} along with either the [Docker CLI](https://www.docker.com/products/cli/){target=\_blank} or [Docker Desktop](https://www.docker.com/products/docker-desktop/){target=\_blank}
```

Comment:
I think this is not needed.

Reply:
Maybe we just include Docker, and remove the rest?